samples/Constraint/availability_constraint.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<mes:Structure xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xml="http://www.w3.org/XML/1998/namespace" xmlns:mes="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/message" xmlns:str="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/structure" xmlns:com="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/common" xsi:schemaLocation="http://www.sdmx.org/resources/sdmxml/schemas/v3_1_0/message ../../schemas/SDMXMessage.xsd">
+	<mes:Header>
+		<mes:ID>IREF583870</mes:ID>
+		<mes:Test>false</mes:Test>
+		<mes:Prepared>2024-07-30T16:53:05Z</mes:Prepared>
+		<mes:Sender id="Unknown"/>
+		<mes:Receiver id="not_supplied"/>
+	</mes:Header>
+	<mes:Structures>
+		<str:AvailabilityConstraints>
+			<str:AvailabilityConstraint>
+				<com:Annotations>
+					<com:Annotation id="obs_count">
+						<com:AnnotationTitle>3299680</com:AnnotationTitle>
+						<com:AnnotationType>sdmx_metrics</com:AnnotationType>
+					</com:Annotation>
+                </com:Annotations>
+				<str:ConstraintAttachment>
+					<str:Dataflow>urn:sdmx:org.sdmx.infomodel.datastructure.Dataflow=ECB:EXR(1.0)</str:Dataflow>
+				</str:ConstraintAttachment>
+				<str:CubeRegion>
+					<str:Component id="CURRENCY_DENOM">

```

Comment:
Finally, `Keyvalue` was not replaced by `Component` thus this should be reverted.

Reply:
As above